### DISCUSSION TITLE
Being able to fork a project from the github app

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#107537] [Mobile]
### Select Topic Area

Product Feedback

### Body

It would be nice to be able to fork a project from the github app android.

**Comment by alcere** [comment]
Ho @Angelk90! Thank you for sharing your feedback! We have shared your request with our product team and it will be taken into consideration for the future, so please stay tuned!

**Reply by Angelk90** [reply]
@alcere : I hope it doesn't take too long to find out some information, for anything tag me and I will support you as I can.

**Reply by alcere** [reply]
Hi @Angelk90! I am happy to share that now repository forking is available for mobile users! Please take a look to our [changelog](https://github.blog/changelog/2024-12-11-now-you-can-fork-public-repositories-on-github-mobile/) to learn more about the feauture!

**Reply by alcere** [reply]
@dpewaal-iba you can find it here: https://github.blog/changelog/2024-12-11-now-you-can-fork-public-repositories-on-github-mobile/

**Comment by jsoref** [comment]
#3333 asked for this. 

https://twitter.com/github/status/1866883185742622746 claims the feature exists

**Comment by Charlesnorris509** [comment]
The ability to fork repositories directly from the GitHub mobile app is indeed a valuable feature that brings convenience to users who prefer mobile workflows. If you're curious about the details of the implementation, the changelog page provides insights into how forking works on mobile and any related updates. You can even send Pull request and Process to testing and Merging through the mobile version
